### PR TITLE
test(tabs): delay story due to potential timing chromatic diff

### PR DESCRIPTION
**Pull request by benelan**
**Related Issue:** #5284

## Summary
Adds a delay to the story because it registered a chromatic diff in what seems like a timing issue
https://www.chromatic.com/test?appId=6266d45509d7eb004aa254fb&id=63e30231eac9ea4af6de885d

### CODE REVIEW

File: src/components/tabs/tabs.stories.ts [hunk 1]
```diff
@@ -174,3 +174,6 @@ export const TabChilrenWithPercentageHeights = (): string => html`
     </calcite-tab>
   </calcite-tabs>
 `;
+TabChilrenWithPercentageHeights.parameters = {
+  chromatic: { delay: 1000 }
```
Comment (jcfranco) [inline src/components/tabs/tabs.stories.ts:178]: Sidebar: I think we could extract this delay in a helper module for reuse.

**Review by jcfranco — APPROVED** [review]

File: src/components/tabs/tabs.stories.ts (hunk 1 above)
Comment (benelan) [reply src/components/tabs/tabs.stories.ts:178]: Yeah, I have it sprinkled around on other stories too. The only thing is I'd rather not delay all stories, only the ones we need to

File: src/components/tabs/tabs.stories.ts [hunk 2]
```diff
@@ -174,3 +174,6 @@ export const TabChilrenWithPercentageHeights = (): string => html`
     </calcite-tab>
   </calcite-tabs>
 `;
+TabChilrenWithPercentageHeights.parameters = {
```
Comment (eriklharper) [inline src/components/tabs/tabs.stories.ts:177]: Should this be spelled `Children`? ☝️

**Comment by eriklharper** [comment]
Issue https://github.com/Esri/calcite-components/issues/5284 doesn't seem related to this PR.  Is that intentional?

File: src/components/tabs/tabs.stories.ts (hunk 2 above)
Comment (jcfranco) [reply src/components/tabs/tabs.stories.ts:177]: `Chilrun` 😂 Good catch.

File: src/components/tabs/tabs.stories.ts (hunk 1 above)
Comment (jcfranco) [reply src/components/tabs/tabs.stories.ts:178]: I meant extracting the delay value, so we would import the same reference.

Comment (benelan) [reply src/components/tabs/tabs.stories.ts:178]: Gotcha. I've been putting different delay values at this point to see what works without eating too much time. But once I figure that out I'll create a const

**Pull request merged**